src/JuliusSweetland.OptiKey/Services/KeyboardOutputService.cs

```diff
@@ -371,6 +371,13 @@ public void ProcessSingleKeyText(string capturedText)
             }
         }
 
+        public void ProcessSingleKeyPress(string key, KeyValuePress.KeyPressType type, int delayMs = 0)
+        {
+            // TODO: This is a stub for future use
+            throw new NotImplementedException();            
```

Comment:
Could you pls test that this is caught and displayed as an error popup. If so could you add the text "This feature is not currently supported"

Reply:
Good spot, this is not caught. Shall I just remove the not-implemented processing stubs, or also the KeyPressKeyValue classes?

Reply:
I've removed the calling code. We now only process ChangeKeyboardKeyValues and classical KeyValues.